conan/cli/commands/config.py

```diff
@@ -28,7 +28,7 @@ def config_install(conan_api, parser, subparser, *args):
                               help="Allow insecure server connections when using SSL. "
                                    "Equivalent to --verify-ssl=False",
                               dest="verify_ssl")
-    subparser.add_argument("-t", "--type", choices=["git", "dir", "file", "url"],
+    subparser.add_argument("-t", "--type", choices=["git", "dir", "file", "url", "pkg"],
```

Comment:
This has potential to have more arguments: ``--remotes`` to specify one or more remotes, ``--lockfile-out`` to generate lockfiles, ``--profiles`` to install different configuration for different platforms (Win, Linux, OSX). 
Adding these arguments to ``conan config install`` will pollute the args namespace for just 1 type. 

Maybe we want a ``conan config install-pkg`` subcommand instead?